src/test/resources/integration/update/update-udt-config.xml

```diff
@@ -31,6 +31,7 @@
     </flow>
 
     <flow name="updatesWithObject">
+        <try transactionalAction="ALWAYS_BEGIN">
```

Comment:
I'm curious, why did you have to add this inside a transaction?

Reply:
Noted, no idea why this fixes it. Will analyze it further and send it again when I figure it out